02_exercise/README.md

```diff
@@ -0,0 +1,16 @@
+# Exercise 1
```

Comment:
Change to correct number

Reply:
I also change the number and description

Reply:
Where??

Reply:
Next comit #2